networks/motion/CNN2D/2D_CNN.py

```diff
@@ -0,0 +1,166 @@
+import os.path
```

Comment:
deprecated file 2D_CNN -> rename to CNN2D
Is this your own file or does this resemble e.g. motion_all_CNN2D?

Reply:
still deprecated, please rename!